cmake_modules/options.cmake

```diff
@@ -9,6 +9,7 @@ option(DEVEL "Enable the building/inclusion of a Development component." OFF)
 option(CLIENTTOOLS_ONLY "Enable the building of Client Tools only." OFF)
 option(INCLUDE_PLUGINS "Enable the building of platform and all plugins for testing purposes" OFF)
 option(USE_CASSANDRA "Include the Cassandra plugin in the base package" ON)
+option(USE_ECLBLAS "Include the ECLBLAS plugin in the base package" ON)
```

Comment:
eclblas was included in the base package already, so I made it work similar to cassandra so there's no confusion in the future.